void main() {
  try {
    //에러가 없을 때 실행할 로직
    final String name = '코드팩토리';

    print(name); //1에러가 없으니 출럭됨
  } catch (e) {
    //catch
  }
}
